.github/workflows/release_publish_docker-image.yml

```diff
@@ -56,4 +56,17 @@ jobs:
           labels: |
             org.opencontainers.image.source=${{ github.event.repository.html_url }}
             org.opencontainers.image.created=${{ steps.prep.outputs.created }}
-            org.opencontainers.image.revision=${{ github.sha }}
\ No newline at end of file
+            org.opencontainers.image.revision=${{ github.sha }}
+      -
+        name: Build and push debian image with journalctl
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile.debian
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: ${{ steps.prep.outputs.tags }}-debian
```

Comment:
@he2ss is the tagging correct here for pushing the debian image?

Reply:
yep, seems a good tag name. :+1: .